modules/infra/control/port.c

```diff
@@ -448,10 +448,10 @@ static int port_mac_add(struct iface *iface, const struct rte_ether_addr *mac) {
 	for (i = 0; i < filter->count; i++) {
 		if (rte_is_same_ether_addr(&filter->mac[i], mac)) {
 			LOG(DEBUG,
-			    "%s: %s mac " ETH_ADDR_FMT " already filtered (refs=%u)",
+			    "%s: %s mac %2p already filtered (refs=%u)",
```

Comment:
instead of using %np all around into the code, which will lead to maintenance complexity, let's keep a FMT with the proper size. For instance:

ETH_ADDR_FMT "%2p"

and same for the other cases

So only those FMT will need to be updated if we come with something different some days.

Reply:
I have added back defines for these printf specifiers. I only shortened them to avoid logs spanning over multiple lines as much as possible. The idea was to make the code more readable.